packages/osd-optimizer/limits.yml

```diff
@@ -74,7 +74,6 @@ pageLoadAssetSize:
   telemetry: 91832
   telemetryManagementSection: 52443
   tileMap: 65337
-  timeline: 29920
```

Comment:
out of curiosity - what does this file even do?

Reply:
I think this limits.yml is a configuration file that is part of the build process, specifically related to the OSD optimizer. This file is used to specify limits for the size of individual plugins in terms of their compiled and bundled JavaScript file sizes. `timeline: 29920` is used for timeline app and `visTypeTimeline: 51933` is for vis_type_timeline. Therefore, we should clean it out.